### DISCUSSION TITLE
Hooks for Blocks

### DISCUSSION THREAD

**Discussion by revnelson** [payloadcms/payload#5239] [Feature Requests & Ideas]
While changing around fields for blocks, I found that hooks don't exist for blocks, and even hooks on the fields in blocks aren't working. Because I'm using some blocks across multiple different collection types, it isn't a good DX to go add hooks to each field that calls each block, and most are even `richText` fields using the blocks feature, which further complicates getting at the block fields within currently usable hooks.

**Comment by unonweb** [comment]
Hi revnelson, generally there're hooks for blocks as specified in the docs. They just don't work within Lexical Editor at the moment:
https://github.com/payloadcms/payload/discussions/4351
